src/pages/code-for-good.js

```diff
@@ -183,6 +181,13 @@ const StyledDashboardCaption = styled(P3)`
 
 const DashboardCaption = StyledDashboardCaption.withComponent('figcaption');
 
+const HeaderActionsContainer = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    padding-top: ${size.large};
```

Comment:
This also allows us to put a basic space between the actions and the above element, then add a bit more which gives spacing between elements when the wrap happens.